### PR TITLE
change `ads`->`kernel-native` and `workload`->`duel-engine` in code

### LINKED ISSUE #937: Replace `ads` and `workload` in the code
**What would you like to be added**:
Replace ads with workload in the code.
**Why is this needed**:
Kmesh advertises that it no longer uses `ads` and `workload`, but uses `duel engine` and `kernel native`, but we still use `ads` and `workload` in our code. should we replace them?

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #937 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
What is the conclusion from community meeting?

**Comment by LiZhenCheng9527** [comment]
I think changing the functions name can be a `good first issue`.
New developers can through this to learn about Kmesh's function capabilities.

File: ctl/dump/dump.go [hunk 1]
```diff
@@ -37,12 +37,12 @@ var log = logger.NewLoggerScope("kmeshctl/dump")
...
-		Example: `# Ads mode:
-kmeshctl dump <kmesh-daemon-pod> ads
+		Short: "Dump config of kernel-native or duel-engine mode",
+		Example: `# Kernel Native mode:
+kmeshctl dump <kmesh-daemon-pod> kernel-native
 	  
-# Workload mode:
-kmeshctl dump <kmesh-daemon-pod> workload`,
+# Duel Engine mode:
+kmeshctl dump <kmesh-daemon-pod> duel-engine`,
```
Comment (hzxuzhonghu) [inline ctl/dump/dump.go:45]: typo

File: ctl/dump/dump.go [hunk 2]
```diff
@@ -54,8 +54,8 @@ kmeshctl dump <kmesh-daemon-pod> workload`,
 func RunDump(cmd *cobra.Command, args []string) error {
 	podName := args[0]
 	mode := args[1]
-	if mode != "ads" && mode != "workload" {
-		log.Errorf("Error: Argument must be 'ads' or 'workload'")
+	if mode != "kernel-native" && mode != "duel-engine" {
```
Comment (hzxuzhonghu) [inline ctl/dump/dump.go:57]: can define a const instead of hardcoding

File: deploy/charts/kmesh-helm/values.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ deploy:
       tag: latest
     imagePullPolicy: IfNotPresent
     containers:
-      kmeshDaemonArgs: "--mode=workload --enable-bypass=false --enable-bpf-log=true"
+      kmeshDaemonArgs: "--mode=duel-engine --enable-bypass=false --enable-bpf-log=true"
```
Comment (hzxuzhonghu) [inline deploy/charts/kmesh-helm/values.yaml:10]: typo

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**